src/main/java/Duke.java

```diff
@@ -1,10 +1,118 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("Good day. YodaBot, I am.");
+        System.out.println("Assistance, you need?");
+        Scanner scanner = new Scanner(System.in);
+        Task[] taskArray = new Task[100];
+        int taskIndex = 0;
+        boolean isEnd = false;
+        String done;
+        while (true) {
+            String command = scanner.nextLine(); //reads in input
+            switch (command) {
+            //if command is list, either display empty or display list
+            case "list":
+                if (taskIndex == 0) {
+                    System.out.println("Empty, list is.");
+                } else {
+                    System.out.println("As shown, list is:");
+                    for (int j = 0; j < taskIndex; ++j) {
+                        if (taskArray[j] instanceof ToDo) {
+                            System.out.println(j + ". " + ((ToDo) taskArray[j]).getToDo() + " " + taskArray[j].getDoneStatus() + " " + taskArray[j].getDescription());
+                        }
+                        if (taskArray[j] instanceof Deadline) {
+                            System.out.println(j + ". " + ((Deadline) taskArray[j]).getDeadline() + " " + taskArray[j].getDoneStatus() + " " + taskArray[j].getDescription() + " (" + ((Deadline) taskArray[j]).getDate() + ")");
+                        }
+                        if (taskArray[j] instanceof Event) {
+                            System.out.println(j + ". " + ((Event) taskArray[j]).getEvent() + " " + taskArray[j].getDoneStatus() + " " + taskArray[j].getDescription() + " (" + ((Event)taskArray[j]).getStartAndEnd() + ")");
+                        }
+                    }
+                }
+                break;
+            //if command is mark, mark the appropriate entry in the list
+            case "mark":
+                System.out.println("Mark what, I ask?");
+                String listIndex = scanner.nextLine();
+                int indexToMark = Integer.parseInt(listIndex);
+                if (taskIndex == 0 || indexToMark > taskIndex) {
+                    System.out.println("In list, it is not.");
+                } else {
+                    taskArray[indexToMark].isDone = true;
+                    System.out.println("Marked it, I have:");
+                    if (taskArray[indexToMark] instanceof ToDo) {
+                        System.out.println(indexToMark + ". " + ((ToDo) taskArray[indexToMark]).getToDo() + " " + taskArray[indexToMark].getDoneStatus() + " " + taskArray[indexToMark].getDescription());
```

Comment:
Would it be better to synthesize the output within the constructor instead of the main file?